src/Evolution/Systems/ScalarAdvection/Subcell/TciOnFdGrid.hpp

```diff
@@ -0,0 +1,45 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/Tags.hpp"
+#include "Evolution/DgSubcell/Tags/Inactive.hpp"
+#include "Evolution/Systems/ScalarAdvection/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+template <size_t Dim>
+class Mesh;
+/// \endcond
+
+namespace ScalarAdvection::subcell {
+/*!
+ * \brief Troubled-cell indicator applied to the finite difference subcell
+ * solution to check if the corresponding DG solution is admissible.
+ *
+ * Applies the Persson TCI to \f$U\f$ on the DG grid.
+ *
+ * \note TCI is run after reconstructing solution to the DG grid during the
```

Comment:
when you rebase this after the other PR is merged can you update `reconstructing solution` to `reconstructing the solution`?

Reply:
I'm not seeing this. Did you forget to push this change maybe?

Reply:
@nilsdeppe My bad :(

Reply:
Oh I've done that a lot :) Just wanted to check :)